### ISSUE TITLE
Logs are not working via the kubectl command

### ISSUE THREAD

**Issue by wolfeidau** [coreos/coreos-kubernetes#59]
I have deployed a rethinkdb to the vagrant based single VM cluster and I am getting the following error when i try and retrieve the logs.

```
$ kubectl logs rethinkdb-rc-751i0
Internal Error: Unrecognized input header
```

It may be related to how the docker is configured to send logs to journald.

**Comment by brosner** [comment]
I am seeing this same issue. Here's a little more information. I ran `docker logs` directly on the master node and got this:

```
# docker logs 89e4e19c84fd
"logs" command is supported only for "json-file" logging driver (got: journald)
```

**Comment by polvi** [comment]
Yes this is related to coreos/bugs#472, coreos/bugs#483. This will be fixed in the next alpha. 

https://github.com/coreos/coreos-overlay/pull/1553#issuecomment-142045751

**Comment by bcwaldon** [comment]
I can confirm that this is addressed in the latest Alpha (815.0.0, https://coreos.com/releases/#815.0.0)
